devmode-start.adoc

```diff
@@ -0,0 +1,6 @@
+Start Open Liberty in development mode, which starts the Open Liberty server and listens for file changes:
+
+[role=command]
+```
+mvn liberty:dev
+```
```

Comment:
If all are good, can add the follow section to this file as well: 

After you see the following message, your application server in development mode is ready.

[role="no_copy"]
----
Press the Enter key to run tests on demand.
----

The development mode holds your command prompt to listen for file changes.
You need to open another command prompt to continue, or simply open the project in your editor.